code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -683,19 +680,48 @@
 /datum/reagent/consumable/maltodextrin
 	name = "Maltodextrin"
 	description = "A common filler found in processed foods. Foods containing it will leave you feeling full for a much shorter time."
-	color = "#ffffff"
+	color = "#d4e1ee"
 	chem_flags = CHEMICAL_RNG_GENERAL
 	taste_mult = 0.1 // Taste the salt and sugar not the cheap carbs
 	taste_description = "processed goodness"
 	nutriment_factor = 0
```

Comment:
```suggestion
	nutriment_factor = -0.3
```

This didn't get saved with the others, this is what I meant by setting nutriment value to a negative.